src/Model/Block.php

```diff
@@ -61,4 +61,21 @@ public function addNode(Object $node): Block
         $this->nodes[] = $node;
         return $this;
     }
+
+    public function getText(): string
+    {
+        $text = '';
+        foreach ($this->nodes as $node) {
+            if ($node instanceof Text) {
```

Comment:
So, we know that a `Block` can only hold `Node` or `Text` instances from the Slate documentation. Let's annotate `Block->getNodes()` with `@return Node[]|Text[]`, and then we won't need to add `instanceof` checks. Especially since the code inside them is identical.